app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallViewModel.kt

```diff
@@ -80,42 +96,14 @@ class InitiatingCallViewModel @Inject constructor(
         )
     }
 
-    private suspend fun initializeScreenState() {
-        conversationDetails(conversationId = conversationId)
-            .collect {
-                callInitiatedState = when (it) {
-                    is ConversationDetails.Group -> {
-                        callInitiatedState.copy(
-                            conversationName = getConversationName(it.conversation.name),
-                            conversationType = ConversationType.Conference
-                        )
-                    }
-                    is ConversationDetails.OneOne -> {
-                        callInitiatedState.copy(
-                            conversationName = getConversationName(it.otherUser.name),
-                            avatarAssetId = UserAvatarAsset(it.otherUser.completePicture ?: ""),
-                            conversationType = ConversationType.OneOnOne
-                        )
-                    }
-                    is ConversationDetails.Self -> throw IllegalStateException("Invalid conversation type")
-                }
-            }
-    }
-
     private suspend fun initiateCall() {
         startCall(
             conversationId = conversationId,
-            conversationType = callInitiatedState.conversationType
+            conversationType = conversationType.await()
```

Comment:
not sure if we need to do an await on global conversation type ? we could just fire a method that returns the conversationtype ?